datalad/cmdline/tests/test_formatters.py

```diff
@@ -19,7 +19,7 @@
 )
 try:
     import formatters as fmt
-except ImportError:  # pragma: no cover
+except ImportError or ModuleNotFoundError:  # pragma: no cover
```

Comment:
Rather than that, the entire `try-except` block could become:

```
skip_if_no_module('formatters')
```

see for example `datalad/downloaders/tests/test_s3.py` for a similar usage.

Reply:
thanks!